tests/src/main/java/org/hibernate/beanvalidation/tck/tests/metadata/ParameterDescriptorTest.java

```diff
@@ -39,14 +42,7 @@
 	@Deployment
 	public static WebArchive createTestArchive() {
 		return webArchiveBuilder()
-				.withTestClass( ParameterDescriptorTest.class )
-				.withClasses(
-						Account.class,
-						Customer.class,
-						CustomerService.class,
-						Executables.class,
-						Person.class
-				)
+				.withTestClassPackage( ParameterDescriptorTest.class )
```

Comment:
Why do you think it's better this way? That package has many other types not needed for this test.